semantic error: Unhashable type: 'list'
 --> tests/errors/test_dict_key4.py:2:12
  |
2 |     print({[1, 2]: "first", [3, 4]: "second"})
  |            ^^^^^^ Mutable type 'list' cannot become a key in dict. Hint: Use an immutable type for key.
